libs/azure-iot-middleware-freertos/source/azure_iot_adu_client.c

```diff
@@ -371,7 +371,7 @@ AzureIoTResult_t AzureIoTADUClient_SendResponse( AzureIoTADUClient_t * pxAzureIo
 }
 
 static void prvFillBaseDeviceInformation( AzureIoTADUClientDeviceInformation_t * pxDeviceInformation,
```

Comment:
maybe do the middleware one too?

Reply:
Yes, there were a bunch of vars and structs left out of the first set. I updated the name (hopefully) everywhere else that was applicable.